testdata/dnn/download_models.py

```diff
@@ -952,6 +952,26 @@ def get_confirm_token(response):  # in case of large files
         url='https://drive.google.com/uc?export=dowload&id=1ClK9WiB492c5OZFKveF3XiHCejoOxINW',
         sha='12ff8b1f5c8bff62e8dd91eabdacdfc998be255e',
         filename='onnx/models/face_recognizer_fast.onnx'),
+    Model(
+        name='wechat_qr_detect',
+        url='https://raw.githubusercontent.com/WeChatCV/opencv_3rdparty/wechat_qrcode/detect.prototxt',
+        sha='a6936962139282d300ebbf15a54c2aa94b144bb7',
+        filename='wechat/detect.prototxt'),
+    Model(
+        name='wechat_qr_detect',
```

Comment:
> 'wechat_qr_detect'

Avoid duplicate names for different entries. Use this name as prefix only.

Reply:
fixed